Revoke-Obfuscation.psm1

```diff
@@ -1047,6 +1051,9 @@ http://www.leeholmes.com/blog/
         [PSTypeName("Microsoft.Management.Infrastructure.CimInstance#root/cimv2/Win32_NTLogEvent")]
         $CimInstance,
         
+        [Parameter(Position = 0, Mandatory = $true, ValueFromPipeline = $false, ParameterSetName = 'Helix')]
+        $Helix,
```

Comment:
If Helix always returns JSON, should this be called HelixJson, and of type 'String'? Then, this code could do the conversion for them. Or alternatively, it could be HelixObject that represents the object parsed out of JSON - for example if they called the Invoke-RestMethod API (which auto-parses JSON)

Reply:
renamed to $HelixObject